app/main.cpp

```diff
@@ -719,6 +719,20 @@ int main( int argc, char *argv[] )
 #endif
   engine.rootContext()->setContextProperty( "__use_simulated_position", use_simulated_position );
 
+  // show "new look and feel" welcome dialog on start?
+  QString lastAppVersion = as.appVersion();
```

Comment:
This lastAppVersion is updated somewhere else I suppose?

Reply:
yeah, it is updated ~50 lines later with `as.setAppVersion( version );`